### ISSUE TITLE
Plotting multiple corrplots on same pave

### ISSUE THREAD

**Issue by cwilso6** [taiyun/corrplot#144]
I would like to be able to plot multiple corrplots on the same page.  Is there a recommended way to plot them using ggpubr or gridExtra packages, preferably ggpubr. Below is an example:
```{r}
library(ggpubr)
data(mtcars)
M <- cor(mtcars)
set.seed(0)

##  different color series
col1 <- colorRampPalette(c("#7F0000", "red", "#FF7F00", "yellow", "white",
                           "cyan", "#007FFF", "blue","#00007F"))
col2 <- colorRampPalette(c("#67001F", "#B2182B", "#D6604D", "#F4A582",
                           "#FDDBC7", "#FFFFFF", "#D1E5F0", "#92C5DE",
                           "#4393C3", "#2166AC", "#053061"))
col3 <- colorRampPalette(c("red", "white", "blue"))
col4 <- colorRampPalette(c("#7F0000", "red", "#FF7F00", "yellow", "#7FFF7F",
                           "cyan", "#007FFF", "blue", "#00007F"))
wb <- c("white", "black")

par(ask = TRUE)

## different color scale and methods to display corr-matrix
plot1=corrplot(M, method = "number", col = "black", cl.pos = "n", title='cutoff')

ggarrange(plot1,plot1)
```
Which produces the following error:
```{r}
Warning messages:
1: In as_grob.default(plot) :
  Cannot convert object of class matrix into a grob.
2: In as_grob.default(plot) :
  Cannot convert object of class matrix into a grob.
```

Again, I want to emphasize that my main goal is to use ggpubr or gridExtra

**Comment by cwilso6** [comment]
Is it possible to have both the matrix and the plot be output from corrplot? As of now, when I assign the plot to a variable it is a matrix. If I could access the plot object, then I think would able to use the desired packages.

**Comment by caijun** [comment]
You can try my R package [ggcorrplot2](https://github.com/caijun/ggcorrplot2), which currently implemented a subset of features of corrplot using ggplot2. Thus, you can combine multiple corrplots into one plot using [cowplot](https://github.com/wilkelab/cowplot) or [patchwork](https://github.com/thomasp85/patchwork). See the following figure.

![image](https://user-images.githubusercontent.com/2522593/70857053-f8b23880-1f22-11ea-8fa0-8cdbd25df05d.png)

**Comment by jukicivan** [comment]
@caijun Your package looks great. However, for some reason, I can't install it. I tried various things but I keep getting "not available for 3.6.1..."

**Comment by caijun** [comment]
I haven't released **ggcorrplot2** on CRAN, thus you are unable to install **ggcorrplot2** via `install.packages("ggcorrplot2")` . Try to install **ggcorrplot2** from github:

```
if (!requireNamespace("devtools")) install.packages("devtools")
devtools::install_github("caijun/ggcorrplot2")
```

**Comment by jukicivan** [comment]
Yes, of course, I realised that. I tried through install_github but then I stumbled upon api problem. This is apparently related to my corporate computer because it now worked on my private mac. However, I still have some issues as it seems that ggcorrplot cant recognise some of the functions that it contains. For example, title, hc.order or lab.

Error in ggcorrplot(cor2, type = "upper", p.mat = p.mat2, hc.order = TRUE,  : 
  unused arguments (hc.order = TRUE, title = "Females")

**Comment by caijun** [comment]
**ggcorrplot2** currently has implemented only a subset of features that **corrplot** has. The parameters `hc.order` and `title`  are not supported yet. Please see the document of `ggcorrplot()`.

**Comment by ErickChacon** [comment]
With `patchwork` and `gridGraphics` you can plot multiple corrplots easily with the function `wrap_elements`. You can do something like:

```r
p1 = wrap_elements(~corrplot(M, method = "number", col = "black", cl.pos = "n", title='cutoff'))
p2 = wrap_elements(~corrplot(M))

p1 + p2
```

See further details at https://patchwork.data-imaginist.com/articles/guides/assembly.html.
